docs/using_eessi/setting_up_environment.md

```diff
@@ -1,6 +1,48 @@
 # Setting up your environment
 
-To set up the EESSI environment, simply run the command:
+To set up the EESSI environment, you can either load an EESSI module with Lmod or source a bash initialisation script.
+
+## Loading an EESSI module with Lmod
+
+There are a few different scenarios where you may want to set up the EESSI environment by loading an EESSI module with Lmod:
+
+1.  You are already using Lmod with version >= 8.6
+
+    In this case, we recommend unsetting the `MODULEPATH`, because EESSI is not designed to mix modules coming from EESSI and from your system.
```

Comment:
```suggestion
    In this case, we _recommend_ unsetting the `MODULEPATH`, because EESSI is not designed to mix modules coming from EESSI and from your system.
```